### PR TITLE
Adding Example UML to SEP 055 and Fixing Typos

### PR DESCRIPTION
**Pull request by nroehner**
These changes should not change any of the best practices described in SEP 055 and rather should clarify them.

### CODE REVIEW

**Review by jakebeal — CHANGES_REQUESTED** [review]
Looking good; just a few changes to request.

**Comment by jakebeal** [comment]
OK, just waiting for the markdown file updates now.

File: sep_055.md [hunk 1]
```diff
@@ -115,19 +113,19 @@ _Example: BBa\_E1010 is a unitary part for an RFP reporter. The BBa\_E1010 part,
...
-Note that this implies that a composite part can be converted to a unitary part by removing its assembly plan (and vice versa, in cases where that is possible).
+A composite part is represented by a `Component` that has its `prov:wasGeneratedBy` property set to a `prov:Activity` representing a RECOMMENDED assembly plan (see Assembly below).
+Note that this implies that a composite part can be converted to a unitary part by removing its assembly plan (and vice versa, for cases in which that is possible).
 Likewise, since `prov:wasGeneratedBy` can have multiple values, a composite part MAY record more than one possible alternative assembly plan.
 
-An assembled part within a composite part SHOULD be represented by the composite `Component` including a `feature` that is a `SubComponent` with an `instanceOf` proprerty linking to the part that has been included by assembly.
+An assembled part within a composite part SHOULD be represented by including a `feature` in the composite `Component` that is a `SubComponent` with an `instanceOf` property that links to the `Component` for the part being assembled.
```
[the remark this answers is not in the document]
Comment (nroehner) [reply sep_055.md]: Reverted to previous phrasing.

File: sep_055.md [hunk 2]
```diff
@@ -230,28 +228,83 @@ A generic assembly template can be generated as a `Component` with digestion and
...
+
+![diagram_abstractions](images/sep_055_diagram_abstractions.png "Common visual abstractions in UML diagrams")
+
+**Figure 1:** Common visual abstractions in UML diagrams, including **(A)** representation of a `Constraint` between two `Features` (in this case, `SubComponents`), **(B)** representation of a `Constraint` between two `ComponentReferences` to `Features` (in this case, `SequenceFeatures`), and **(C)** representation of two different types of `Location`.
+
+#### Composite Part <a name="assembly"></a>
+
+![composite_part](images/sep_055_composite_part_example_1.png "Composite part example UML diagram with Locations")
+
+**Figure 2:** Composite part example UML diagram with `Locations`. SO:0001953 stands for restriction_enzyme_assembly_scar and SO:0000139 stands for ribosome_entry_site.
```
[the remark this answers is not in the document]
Comment (nroehner) [reply sep_055.md]: Example UML diagrams now have narrative description.

**Comment by nroehner** [comment]
I think this should be good to merge unless you meant something else by markdown file updates.

**Pull request merged**